clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -131,10 +131,10 @@ public DescribeTopicsResults describeTopics(Collection<String> topicNames) {
     }
 
     /**
-     * Descripe an individual topic in the cluster.
+     * Describe some topics in the cluster.
      *
      * Note that if auto.create.topics.enable is true on the brokers,
-     * AdminClient#describeTopic(topicName) may create a topic named topicName.
+     * describeTopics(topicName, ...) may create a topic named topicName.
```

Comment:
Hmm.  This is intended to point out that 'topicName' is part of the collection of topics being described.  It's not a reference to a different function.